### PR TITLE
docs(aio): add Adam Plumer to Angular collaborators

### PR DESCRIPTION
**Pull request by alexeagle**
/cc @CaerusKaru

### CODE REVIEW

File: aio/content/marketing/contributors.json [hunk 1]
```diff
@@ -643,5 +643,11 @@
     "website": "https://medium.com/@zizzamia",
     "bio": "Leonardo is a Senior Software Engineer at Coinbase. He is deeply passionate about web performance and most recently developed Perfume.js to help companies prioritize roadmaps and business, through performance analytics. From 2018, co-organizer of the Angular San Francisco Meetup group and NGRome Conference.",
     "group": "GDE"
+  },
+  "CaerusKaru": {
+    "name": "Adam Plumer",
+    "group": "Collaborator",
+    "mentor": "vikerman",
+    "picture": "CaerusKaru.jpg"
```
Comment (CaerusKaru) [inline aio/content/marketing/contributors.json]:
```suggestion
    "picture": "CaerusKaru.jpg",
    "bio": "Adam is a Software Engineer at Amazon. He is a maintainer for Angular Universal and Flex Layout. In his spare time, Adam enjoys coming up with things to do in his spare time." 
```

**Review by CaerusKaru — APPROVED** [review]
LGTM, thanks!

**Comment by alexeagle** [comment]
We'll need to get a signed NDA from you so we can invite you to meetings. @manughub will send one.

**Comment by alexeagle** [comment]
@manughub says the NDA is signed, thanks!

**Pull request closed without merging**